zerver/lib/bugdown/__init__.py

```diff
@@ -1200,6 +1201,17 @@ def run(self, lines):
                 if re.search(match_re, content):
                     current_message.alert_words.add(word)
 
+            for command in realm_slash_commands:
+                escaped = re.escape(("/" + command).lower())
+                match_re = re.compile(u'(?:%s)%s(?:%s)' %
+                                      (allowed_before_punctuation,
+                                       escaped,
+                                       allowed_after_punctuation))
+                if re.search(match_re, content):
+                    current_message.triggered_slash_commands.add(command)
+            if hasattr(current_message, 'triggered_slash_commands') and \
```

Comment:
I can't remember the exact case, but this was failing on some test cases. It is also possible that because the tests were failing at other locations, I added it here as well. But either way, with our new approach, this step won't be required.